fastddsspy_tool/test/application/tests.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for the fastddsspy executable.
+
+Contains a package of system test for fastddsspy tool
+
+Usage: test.py -e <binary_path>
+
+Arguments:
+
+    Fast DDS Spy binary path          : -e | --exe binary_path
+
+    Run test in Debug mode          : -d | --debug
+
+    Use SIGINT or SIGTERM           : -s | --signal sigint|sigterm
```

Comment:
This argument is no longer required.
And there are some other arguments that could be requested.